chsdi/models/vector/stopo.py

```diff
@@ -697,6 +697,23 @@ class GeolGenKarteGGK200(Base, Vector):
 register('ch.swisstopo.geologie-generalkarte-ggk200', GeolGenKarteGGK200)
 
 
+class GeolGenKarteGGK200Meta(Base, Vector):
+    __tablename__ = 'kv_ggk_pk'
```

Comment:
yes, once for the raster layer tooltip and once for the vector (meta) layer tooltip.